Cases/Create case with jQuery.js

```diff
@@ -2,164 +2,52 @@
 This example shows how to create a new case.
 In this example the following fields are used:
 - Field with id 1: subject field
```

Comment:
Voegen deze velden weinig toe of waarom heb je ze weggehaald?

Reply:
dit was een oud voorbeeld, en dat is niet meer relevant